client/src/components/Notify.re

```diff
@@ -1,6 +1,7 @@
-Modules.require("./Notify.css");
+Modules.import("./Notify.css");
 open Utils;
 
+[@react.component]
 let base = (~className, ~sticky=false, message) =>
```

Comment:
`message` should be a labelled arg. Probably it was meant to be `children` 🤔

Reply:
I changed it like labeled. But according code this variable must be string type, not children. Maybe.